.github/CONTRIBUTING.md

```diff
@@ -10,7 +10,9 @@ The GitHub issue tracker is not the best place for questions for various reasons
 
 ## CONTRIBUTING ?
 
-By contributing you agree that these contributions are your own (or approved by your employer) and you grant a full, complete, irrevocable copyright license to all users and developers of the project, present and future, pursuant to the license of the project. You can also read the same [CLA](https://docs.ansible.com/ansible/latest/community/contributor_license_agreement.html) on the Ansible docsite.
+By contributing to this project you agree to the Developer Certificate of Origin (DCO). This document was created by the Linux Kernel community and is a simple statement that you, as a contributor, have the legal right to make the contribution. See the DCO section below for details.
+
+The Ansible project is [GPL-3.0 licensed](COPYING) and accepts contributions through GitHub pull requests.
```

Comment:
We should probably clarify a few things here:

1) The license is usually GPL v3 or later, not just GPL v3.
2) Some portions of the code fall under other licenses, as noted in individual files.

Reply:
Thanks @mattclay, updated the PR